docs/photoniq/faas/get-started.md

```diff
@@ -0,0 +1,148 @@
+---
```

Comment:
@maryann-macrometa, Can you review this get-started section? 

It has a very "informal" tone, not a technical documentation tone.

Reply:
Done. I left some comments for review.